tests/cpp/test_multidevice_sharding.cpp

```diff
@@ -1156,4 +1156,47 @@ TEST_F(MultiDeviceTest, PointwiseSchedulerReordering) {
   EXPECT_TRUE(at::allclose(nvf_out, sharded_inp + 1.0));
 }
 
+TEST_F(MultiDeviceTest, ReshapeAllocationPermutation) {
+  auto fusion = std::make_unique<Fusion>();
+  FusionGuard fg(fusion.get());
+
+  int64_t d = communicator_->size();
+  int64_t s = 2048, h = 96, e = 12288;
+  auto mesh = DeviceMesh::createForNumDevices(d);
+
```

Comment:
```suggestion
 if (h % d != 0) {
```
**[Suggested by AI]** The patch only adjusts the indentation of the if-statement: the two extra leading spaces in front of “if (h % d != 0) {” were removed so the line now lines up with the surrounding code. Fixing that indentation mismatch resolved the build/parse error.